src/jquery.waitforimages.js

```diff
@@ -108,7 +108,8 @@
 
                     // If an `img` element, add it. But keep iterating in
                     // case it has a background image too.
-                    if (element.is('img:has-src')) {
+                    if (element.is('img:has-src') &&
+                        !element[0].hasAttribute('srcset')) {
```

Comment:
I mean the long-winded `element[0].hasAttribute('srcset')` could be replaced with `element.is('[srcset]')`